lte/gateway/c/oai/tasks/mme_app/mme_config.c

```diff
@@ -679,7 +691,18 @@ int mme_config_parse_file(mme_config_t *config_pP)
       }
 
       config_pP->served_tai.nb_tai = num;
-      AssertFatal(16 >= num, "Too many TAIs configured %d", num);
+      if (num > MAX_TAI_SUPPORTED) {
+        OAILOG_CRITICAL(
+          LOG_CONFIG,
+          "Too many TAIs configured: %d (Maximum supported: %d)\n",
+          num,
+          MAX_TAI_SUPPORTED);
+      }
+      AssertFatal(
```

Comment:
Can we add the OAILOG_CRITICAL call inside the AssertFatal call itself? It will reduce the redundant text here.

Reply:
OAILOG_CRITICAL has been added in the Assert call itself and redundant logs have been removed.